thirdparty/deto_async/async/internal/abstractinvoker.cpp

```diff
@@ -32,6 +33,10 @@ void AbstractInvoker::invoke(int type, const NotifyData& data)
     CallBacks callbacks = it->second;
 
     for (const CallBack& c : callbacks) {
+        if (!it->second.containsReceiver(c.receiver)) {
+            qDebug("Skipping removed receiver");
```

Comment:
For the case callback uninstalls another when an Asyncable is deleted, e.g. during InspectorListModel::removeUnusedModels (called by setElementList, called from a currentNotationChanged callback).